Content/en-us/Face/Face-API-How-to-Topics/HowtoAddFaces.md

```diff
@@ -0,0 +1,128 @@
+<!--
+NavPath: Face API/How-to Topics
+LinkLabel: How to Add Faces
+Url: face-api/documentation/face-api-how-to-topics/howtoaddfaces
+Weight: 220
+-->
+# How to Add Faces
+
+This guide will demonstrate the best practice to add massive number of persons and faces to a person group, which also applies to face lists. The samples are written in C# using the Face API client library.
+
+## Table of Contents
+
+- [Step 1: Initialization](#step1)
+- [Step 1: Authorize the API call](#step1)
+- [Step 2: Create the person group](#step2)
+- [Step 3: Create the persons to the person group](#step3)
+- [Step 4: Add faces to the persons](#step4)
+- [Summary](#summary)
+
+## <a name="step1"></a> Step 1: Initialization
+
+Several variables are declared and a helper function is implemented to schedule the requests.
+
+- `PersonCount` is the toal number of persons.
+- `CallLimitPerSecond` is the maximum calls per second according to the subscription.
```

Comment:
`CallLimitPerSecond` is the maximum calls per second according to the subscription tier.